.github/workflows/greetings.yml

```
@@ -0,0 +1,41 @@
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      # Step 2: Send a personalized greeting
+      - name: Greet new contributors
+        uses: actions/first-interaction@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-message: >
+            👋 Hello @{{ sender.login }}!  
+            Thank you for opening this issue. We appreciate your contribution to the project!  
+            Please review our [contribution guidelines](./CONTRIBUTING.md) if you haven’t already.  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)